src/Components/Server/ref/Microsoft.AspNetCore.Components.Server.netcoreapp3.0.cs

```diff
@@ -103,6 +103,7 @@ public partial interface IServerSideBlazorBuilder
     }
     public static partial class ServerSizeBlazorBuilderExtensions
```

Comment:
I know it predates your PR, but there's a typo in the class name. Server "size"?

Reply:
The typo is also in the filename. Would you mind fixing it as part of this PR?

Reply:
lol. We should change the product to be ServerSizeBlazor 😄